modules/.releng/dxp/apps/portal-search-solr8/app.changelog

```diff
@@ -45,3 +45,15 @@ app.artifacts-5.1.0=\
 app.change.log-5.1.0=LPD-65931 LPD-21872 LPD-22171 LPD-22737 LPD-22945 LPD-22945 LPD-25779 LPD-28411 LPD-28413 LPD-28412 LPD-28416 LPD-28414 LPD-35124 LPD-46638 LPD-56284 LPD-58072 LPD-58182 LPD-58424 LPD-58417 LPD-58530 LPD-58528 LPD-58977 LPD-59060 LPD-59218 LPD-50745 LPD-60412 LPD-60487 LPE-18299
 
 app.git.id-5.1.0=5efe9fba8dd9df53eb413ed01b744c798a83f42e
+
+#
+# Liferay App Version 5.1.1
+#
+
+app.artifacts-5.1.1=\
+	com.liferay.portal.search.solr8.api-5.0.3.jar,\
+	com.liferay.portal.search.solr8.impl-5.0.47.jar
+
+app.change.log-5.1.1=LPD-15847 LPD-18681 LPD-20207 LPD-20777 LPD-21007 LPD-21809 LPD-21872 LPD-22171 LPD-22737 LPD-22945 LPD-23848 LPD-25046 LPD-25779 LPD-28411 LPD-28412 LPD-28413 LPD-28414 LPD-28416 LPD-33240 LPD-35023 LPD-35124 LPD-39064 LPD-41038 LPD-44271 LPD-46473 LPD-46638 LPD-48017 LPD-50745 LPD-52161 LPD-54562 LPD-56284 LPD-58072 LPD-58182 LPD-58417 LPD-58424 LPD-58528 LPD-58530 LPD-58977 LPD-59060 LPD-59064 LPD-59218 LPD-60412 LPD-60487 LPD-61181 LPD-65931 LPD-66826 LPD-7822 LPS-156857 LPS-157425 LPS-166987 LPS-168749 LPS-171270 LPS-171941 LPS-171976 LPS-172192 LPS-172250 LPS-173765 LPS-174196 LPS-174289 LPS-176632 LPS-180562 LPS-180691 LPS-181002 LPS-181508 LPS-181829 LPS-182432 LPS-182523 LPS-183440 LPS-185533 LPS-186952 LPS-187293 LPS-187417 LPS-188507 LPS-188912 LPS-194520 LPS-195459 LPS-196524 LPS-198392 LPS-199025 LPS-199195 LPS-199540 RELEASE-5743
```

Comment:
Hello @nikki-pru 

May I ask you to generate a manual delta by checking the diff of `modules/apps/portal-search-solr8` between the hash of the tag of v5.1.0 and this hash?